tests/flag_attn/test_flash_attention.py

```diff
@@ -29,23 +29,32 @@ def report(name, actual, expected):
     (1, 2, 8192, 16, 10),
     (1, 2, 8192, 32, 0),
 ])
+@pytest.mark.parametrize('bias', [True, False])
 @pytest.mark.parametrize('causal', [True, False])
 @pytest.mark.parametrize('dtype', [torch.float16, torch.bfloat16])
 @pytest.mark.parametrize('stride_order', ['BHTD', 'BTHD'])
-def test_attention_fwd(B, H, T, D, P_SEQ, causal, stride_order, dtype, scale, device_id):
+def test_attention_fwd(B, H, T, D, P_SEQ, bias, causal, stride_order, dtype, scale, device_id):
     device = f"cuda:{device_id}"
     if stride_order == "BHTD":
         q = torch.empty((B, H, T, D), dtype=dtype, device=device).normal_(mean=0., std=scale)
         k = torch.empty((B, H, T + P_SEQ, D), dtype=dtype, device=device).normal_(mean=0., std=scale)
         v = torch.empty((B, H, T + P_SEQ, D), dtype=dtype, device=device).normal_(mean=0., std=scale)
+        if bias: 
+            bias = torch.empty((B, H, T, T+P_SEQ), dtype=dtype, device=device).normal_(mean=0., std=scale)
+        else: 
+            bias = None
     else:
         q = torch.empty((B, T, H, D), dtype=dtype, device=device).normal_(mean=0., std=scale).transpose(1, 2)
         k = torch.empty((B, T + P_SEQ, H, D), dtype=dtype, device=device).normal_(mean=0., std=scale).transpose(1, 2)
         v = torch.empty((B, T + P_SEQ, H, D), dtype=dtype, device=device).normal_(mean=0., std=scale).transpose(1, 2)
+        if bias: 
+            bias = torch.empty((B, T, T+P_SEQ, H), dtype=dtype, device=device).normal_(mean=0., std=scale)
```

Comment:
The shape of bias should always be a shape that is broadcastable to (B, H, Tq, Tk), or it would cause some error related to shape.